specification/automation/resource-manager/Microsoft.Automation/stable/2022-07-15/python3package.json

```diff
@@ -0,0 +1,703 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagement",
+    "version": "2019-06-01",
+    "x-ms-code-generation-settings": {
```

Comment:
The first commit needs to be the previous api version and the new changes should only be added in the subsequent commits. This is explained in detail here (If using Open API which is recommended)  [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version) or here(For manual process) :  [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version) . Please remove the "ARMChangesRequested" tag once you have fixed this issue.

Reply:
Hi rkmanda, Thanks for reviewing. 
I have added first commit with pervious api version swagger and committed my changes to next commit can you please review again.